File: PRAK403-2210817120009-Najah_Maisyaroh.py
bil1, bil2 = map(int, input().split())
if(bil1 < bil2):
    a = 1
    for i in range(bil1, bil2+1):
        x= bil2 - a + 1
        a +=  1
        if (i == bil1):
            print(end="")
        else:
            print(" - ", end=" ")
        print(f"{i} {x}", end=" ")
else:
    a = 0
    for i in reversed(range(bil2, bil1 + 1)):
        x = bil2 + a
        a += 1
        if (i == bil1):
            print(end="")
        else:
            print(" - ", end=" ")
        print(f"{i} {x}", end=" ")

File: PRAK403-2210817120009-Najah_Maisyaroh.c
#include <stdio.h>
int main(){
    int bil1, bil2;
    int x = 0;
    scanf("%d %d", &bil1, &bil2);
    if (bil1 > bil2){
        for (int i = bil1; i >= bil2; i--){
            if (i == bil1){}
            else{
                printf(" - ");}
            printf("%d %d", i, bil2 + x);
            x++;}} 
    else{
        for (int i = bil1; i <= bil2; i++){
            if (i == bil1){}
            else{
                printf(" - ");}
            printf("%d %d", i, bil1+bil2-i);}}  
}